lib/internal/timers.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+// Timeout values > TIMEOUT_MAX are set to 1.
+const TIMEOUT_MAX = 2147483647; // 2^31-1
+
+module.exports = {
```

Comment:
Why? It works fine here, and other files have it like that. I don't see a good reason to change?